src/components/checkbox/README.md

```diff
@@ -1,155 +1,80 @@
 
-This example includes a hidden input in the markup, as some frameworks require. It requires that the input and label be wrapped in a `.rvt-checkbox-wrapper`. Here we are using the wrapper class on a `li`, but it will also work with generic elements like a `div` and `span`.
+The markup in this example will visually hide the label element leaving only the checkbox element visible. This can be useful in situations such as when checkboxes are used to select entire rows of data from a table.
 
+```html
+<span class="rvt-checkbox rvt-checkbox--sr-only-label">
+  <input type="checkbox" name="checkbox-demo" id="checkbox-hidden-label">
+  <label for="checkbox-hidden-label">This label text is visually hidden</label>
+</span>
 ```
-<fieldset>
-    <legend class="rvt-ts-23 rvt-m-bottom-lg">Alternate markup (visually identical)</legend>
-    <ul class="rvt-inline-list">
-        <li class="rvt-checkbox-wrapper">
-            <input type="checkbox" name="checkbox-demo" id="checkbox-9">
-            <input type="hidden">
-            <label for="checkbox-9">Option one</label>
-        </li>
-        <li class="rvt-checkbox-wrapper">
-            <input type="checkbox" name="checkbox-demo" id="checkbox-10">
-            <input type="hidden">
-            <label for="checkbox-10">Option two</label>
-        </li>
-    </ul>
-</fieldset>
-```
-
-## Implementation notes
-
-### Checkbox labels
 
-- Write the label so that it describes the selected state of the checkbox
-- Write the label as a phrase or an imperative sentence, and use no ending punctuation
-        - **Exception:** If a checkbox label also labels a subordinate control that follows it, end the label with a colon
-- Use sentence-style capitalization
-- For a group of checkboxes, use parallel phrasing and try to keep the length about the same for all labels
-- For a group of checkboxes, focus the label text on the differences among the options. If all the options have the same introductory text, move that text to the group label.
-- Use positive phrasing. Don't phrase a label so that selecting a checkbox means not to perform an action.
-        - **Exception: Don't show this again** checkboxes
+## Description Element
+In some cases it can be helpful to provide additional more detailed information about the functionality of a checkbox in addition to the the text inside the `<label>` element. In these instances you can use the `.rvt-checkbox__description` element to provide more context to a user.
 
-### Checkbox group labels
+1. Add an `aria-describedby` attribute with a unique value to the checkbox input
+1. Add a `<div class="rvt-checkbox__description"></div>` element after the <label> element as direct child of the `<span class="rvt-checkbox">` element.
+1. Add an `id` attribute to the description element that matches the `aria-describedby` attribute added in step 1.
```

Comment:
This feels like it could be reworked a little. Try something like this:

```md
1. Add a `<div class="rvt-checkbox__description" id="checkbox-description"></div>` element as a direct child of the `<span class="rvt-checkbox">` element, placed after the `<label>` element. Make the id unique for the document.
2. Add an `aria-describedby` attribute to the checkbox `<input>`, referencing the `id` attribute of the description element.
```

This is an alternate version of the second one. Don't know if it is any better.

```md
2. Add an `<input aria-describedby="checkbox-description">` attribute to the checkbox. Reference the `id` attribute of the description element.
```